packages/xo-server-transport-icinga2/src/index.js

```diff
@@ -0,0 +1,126 @@
+import createLogger from '@xen-orchestra/log'
+import httpRequestPlus from 'http-request-plus'
+
+// =============================================================================
+
+const log = createLogger('xo:xo-server:transport-icinga2')
+
+// =============================================================================
+
+export const configurationSchema = {
+  type: 'object',
+
+  properties: {
+    server: {
+      type: 'string',
+      description: 'The icinga2 server address',
+    },
+    port: {
+      type: 'integer',
+      description: 'The access port',
+    },
+    user: {
+      type: 'string',
+      description: 'The icinga2 server username',
+    },
+    password: {
+      type: 'string',
+      description: 'The icinga2 server password',
+    },
```

Comment:
Why not directly ask for the URL?

What are Icinga2 users used to?

Reply:
idea, i'll ask the user whom requested the feature.

I did it similarly as Nagios